lib/stat_tracker.rb

```diff
@@ -94,5 +94,41 @@ def goals_per_team
     team_goals
   end
 
+  def highest_sum 
+    hash = {}
+    @game_factory.games.each do |i|
```

Comment:
i think this looks great, clay! i think we should consider renaming the block variable to be more explicit as to what it represents and maybe also renaming the hash variable to be more explicit as well. 

maybe you could change the `i` to be `game` instead?